### DISCUSSION TITLE
Failing to compile on ARM device

### DISCUSSION THREAD

**Discussion by jsimmonsworks24** [rustdesk/rustdesk#4412] [Q&A]
Hello! I'm following the guide on how to compile for a Raspberry, since that most closely matches the device I'm using. The compile keeps failing with message:

  --- stderr
  `PKG_CONFIG_ALLOW_SYSTEM_CFLAGS="1" "pkg-config" "--libs" "--cflags" "gstreamer-video-1.0" "gstreamer-video-1.0 >= 1.8"` did not exit successfully: exit status: 1
  error: could not find system library 'gstreamer-video-1.0' required by the 'gstreamer-video-sys' crate

  --- stderr
  Package librga was not found in the pkg-config search path.
  Perhaps you should add the directory containing `librga.pc'
  to the PKG_CONFIG_PATH environment variable
  Package 'librga', required by 'gstreamer-video-1.0', not found

Searched the web and did a find on my device. I don't really know where/what librga.pc is or how to get it. I've installed gstreamer and all the packages related to it that I can find. Any ideas? Thank you!

**Comment by defencedog** [comment]
Same here

**Comment by rustdesk** [comment]
#763
